libs/api/domains/syslumenn/src/lib/client/models/dataUpload.ts

```diff
@@ -14,6 +14,7 @@ export interface IDataUpload {
     skeytaHeiti: string
     adilar: ChildrenTransferPerson[]
     attachments: File[]
+    gagnaMengi: object
```

Comment:
According to the documentation from Þjónustugátt starfskerfa this is a key: string object, should we type it in the same way instead of just `object`?

``` 
{ [key: string]: string }
```

Reply:
True, this was lazy of me 🙈 will fix